cylc/flow/parsec/fileparse.py

```diff
@@ -220,7 +220,7 @@ def multiline(flines, value, index, maxline):
     return quot + newvalue + line, index
 
 
-def process_plugins(fpath):
+def process_plugins(fpath, opts):
     # Load Rose Vars, if a ``rose-suite.conf`` file is present.
```

Comment:
Now done: I considered adding Typing, but decided to leave it for now: Typing this will be more fun once we've dropped 3.7 support: https://www.python.org/dev/peps/pep-0589/